UM/Qt/Bindings/ProfilesModel.py

```diff
@@ -108,18 +117,72 @@ def exportProfile(self, name, url):
         if not profile:
             return
 
-        error = ""
+        #Parse the fileType to deduce what plugin can save the file format.
```

Comment:
It would be nice to not have to deal with the parsing here and instead use a dict lookup or similar to retrieve the plugin that corresponds to a certain file type. This is what I do in LocalFileOutputDevice. For now, you can leave this as is but please add a TODO comment here about this.